settingslogic.gemspec

```diff
@@ -3,7 +3,7 @@ $:.push File.expand_path("../lib", __FILE__)
 
 Gem::Specification.new do |s|
   s.name        = "settingslogic"
-  s.version     = "2.0.9"
+  s.version     = "2.0.10.1"
```

Comment:
Just putting something here so we can still pick a version that may not collide. Let me know if I should do something different. My thinking is that since this isn't an actual tag and is in our branch, we can put whatever we want and still be able to tell in the lock file which "revision" we are using.
This is short term until we decide what we want to do long term with this fork/code.

Reply:
I think this is not the best way, it can be seen as a security threat... I am thinking at this point maybe we do a new repo with a new name and start with version 1.0 ???

Reply:
Remember, this is short term and is not an actual gem release. What I want is a way to easily differentiate this code from the main branch code in the Gemfile.lock and not just in the branch info.  The existing 2.0.9 has been there and does not correspond to a TAG nor an actual release and it has been that way for years. No issues.
We need these changes now and can work on a long term plan later. 
I see no harm in this change, but if you feel strongly, I can leave it as it was, but then we will be using a different version with the same "version" number that shows up in the Gemfile.lock file as existing installs. The only change would be the location change from branch master to branch amco. I.e if I do something like `grep settings Gemfile.lock` it will return something like:
```
      settingslogic
  remote: https://github.com/amco/settingslogic.git
    settingslogic (2.0.9)
  settingslogic!
```
Which doesn't show the branch name. That is why I wanted that second visual reminder.